src/utils/sources.ts

```diff
@@ -129,6 +129,10 @@ export type GetSourceFormValues = {
 export const getSourceFormValues = (entityDetails: Test, testSources: SourceWithRepository[]): GetSourceFormValues => {
   const {content} = entityDetails;
 
+  if (!content || !content.type) {
```

Comment:
```suggestion
  if (!content?.type) {
```

Reply:
In general it shouldn't be acceptable but for some reason in under some circumstances it's not present.